app/models/candidate_interface/personal_details_form.rb

```diff
@@ -30,7 +30,8 @@ def self.build_from_application(application_form)
         last_name: application_form.last_name,
         first_nationality: application_form.first_nationality,
         second_nationality: application_form.second_nationality,
-        english_main_language: application_form.english_main_language ? 'yes' : 'no',
+        english_main_language: application_form.english_main_language.nil? ? nil
+          : application_form.english_main_language ? 'yes' : 'no',
```

Comment:
Nested ternary 😐 — could refactor?

Reply:
Yep... I will come back to this PR. Literally just opened it to get the context out of my head (as I knew how to fix the bug) and only did red + green.